packages/app/src/components/uiBlocks/TagEntry.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div class="flex flex-col flex-shrink-0">
+    <div class="relative pb-1/1">
+      <img class="absolute w-full h-full rounded-xl" :src="image" />
+    </div>
+    <div id="TagName" class="h-2/6 text-white font-semibold" :class="[`text-${textSize}`]">
+      {{ name }}
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent, ref } from 'vue';
+
+export default defineComponent({
+  props: {
+    img: {
+      type: String,
+      required: true,
+      default: '',
+    },
+    name: {
+      type: String,
+      required: true,
+    },
+    textSize: {
+      type: String,
+      required: false,
+      default: 'base',
+    },
+  },
+  setup(props) {
+    const image = ref<string>(`./${props.img}`);
```

Comment:
Maybe a computed would be better suited here. This way if the parent somehow decides to change the image your component would not notice and no update would happen.

Reply:
fixed in a7037769edd10cd00aff27d4e9346a0fbff9111e